soc/nxp/imx/imx8m/m7/soc.c

```diff
@@ -148,6 +148,38 @@ static void SOC_ClockInit(void)
 	CLOCK_EnableClock(kCLOCK_Sec_Debug);
 }
 
+static void gpio_init(void)
+{
+
+#if defined(CONFIG_UART_MCUX_IUART)
```

Comment:
Why is the dependency here on this Kconfig? Shouldn't it be `CONFIG_GPIO_IMX`?

Reply:
Thanks @danieldegrasse , that was a copy paste error. I changed it.

As well I run clang-format for this file and it formatted some other lines of code I didn't change.

I committed the formatting to be compliant with the current clang-format rules.

Reply:
Thanks for running clang-format- if you are going to do this though, please make one commit specifically with the changes introduced by clang-format. That makes it *much* easier to review the PR, and makes it simpler for someone following the commit history of the file to understand the reasons for changes

Reply:
okay, then I will undo the changes regarding clang-format. I see the point to separate the two topics in different PR's.